front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -133,9 +143,21 @@ export default function Project() {
     setIsLoading(false);
   };
 
+  const toggleStudySelection = (id?: number) => {
+    toggleSelection(id, selectedStudyIds, setSelectedStudyIds);
+  };
+
+  const handleDeleteStudy = () => {
+    if (selectedStudyIds && projectId) {
+      selectedStudyIds.forEach((studyId) => deleteStudyV2({ projectId, studyId }));
+      setStudiesList(studiesList.filter((study) => !selectedStudyIds.includes(study.id)));
+      setSelectedStudyIds([]);
+    }
+  };
```

Comment:
I'll leave the global review and decision regarding this to @kmer2016, but I feel like we could have all this logic (which I guess is the same for project/study/scenario) in a custom hook handling the selection and deletion. And maybe we could pass to this hook as parameter the deleteFunction ? 

Like this for example `const { selectedItems } = useHandleMultiSelectionDeletion(deleteScenario)`.

What do you think ?